AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj

```diff
@@ -1926,6 +1928,7 @@
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 0.22.7;
 				DEBUG_INFORMATION_FORMAT = dwarf;
+				ENABLE_BITCODE = YES;
```

Comment:
Are we still keeping it `YES`? It's `NO` in demo.

Reply:
Keeping yes for SDK, no for demo. The reason is the demo has a dependency on Toast which does not build with Bitcode. We want to keep Bitcode for SDK for now to not break customers.

Reply:
The toast has been fixed.